.github/workflows/continuous-integration.yml

```diff
@@ -18,6 +17,13 @@ jobs:
         with:
           args: --from=${{ github.event.pull_request.base.sha }}
 
+  composer_normalize:
+    name: "Composer Normalize"
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: "Composer Normalize"
+        run: "composer normalize --dry-run"
```

Comment:
Here is some help regarding this: https://github.com/marketplace/actions/composer-normalize-action

Reply:
Good help; thanks